platform_versioned_docs/version-24.1/data_studios/managing.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: "Managing studio sessions"
+description: "Data Studios public preview."
+date: "1 Oct 2024"
+tags: [data, studios]
+---
+
+## Add a data studio
+
+This functionality is available to users with the **Maintain** role and above.
+
+To add a new data studio, complete the following steps:
+
+1. In a workspace, select **Data Studios**, and then select **Add data studio**.
+1. For the **Compute & Data** section, complete the following steps:
+
+   1. Customize the following fields:
+      - **Select compute environment**: Only AWS Batch is supported.
+      - **CPUs allocated**: The default allocation is 2 CPUs.
```

Comment:
Regarding the new GPU support, we should add here something along the lines of:

- **GPUs allocated**: This field is only available if the selected Compute Environment has GPU enabled (enabled flag for `Use Amazon-recommended GPU-optimized ECS AMI`). The default allocation is 0 GPUs.